backend/movement/movement.py

```diff
@@ -0,0 +1,254 @@
+from backend.movement.player import Player
+from backend.movement.station import Station
+from enum import Enum
+
+class MetaData(object):
+    def __init__(self, path, time):
```

Comment:
a docstring including information on what path and time represents would be helpful

Reply:
Added documentation